src/types.ts

```diff
@@ -88,6 +88,7 @@ export type NuxtI18nOptions<Context = unknown> = {
   types?: 'composition' | 'legacy'
   debug?: boolean
   dynamicRouteParams?: boolean
+  parallelPlugin: boolean
```

Comment:
```suggestion
  parallelPlugin?: boolean
```
That option should be typed defined above. Because options is optional.